data/gegno/gegno intro missions.txt

```diff
@@ -0,0 +1,331 @@
+			choice
+				`	"Where am I?"`
+			`	The Quarg sits down across from you, still gesturing their arms about, albeit slowly, in a strange matter. "You, brave rover, have found yourself in the dwelling of the mighty Gegno, who have rooted their hardy empire here in the great galactic east." You wonder if the Quarg's use of boastful adjectives is because of gaps in his translations, or if they are intentionally speaking of the Gegno in that way. Either way, they leave an odd impression on you- their voice still comes from the lower tones of Quarg speech, rather than something more excitable accompaying the pace that they are speaking at.`
+			choice
+				`	"The Gegno?"`
+					goto gegno
+				`	"Are the ships I’ve seen these Gegno?"`
+					goto ships
+			label gegno
+			`	"Yes, the imperious Gegno. A juvenile race, they are, many decades younger than that of you humans. They do not find themselves in the struggle of leadership the same way as others do. Rather, they glorify their pursuit of guidance between one another to share a noble future."`
+			choice 
+				`	"What is this guidance they pursue?"`
+				`	"A noble future?"`
+					goto guidance
+			label ships
+			`	"Indeed, the vessels of such stature represent the robust Gegno. Not in sorrow, but greatness do they pursue their future, bearing the fruit of sacrifice as a trophy to their name." The Quarg now shifts facial expressions in such a way to suggest a small hint of frustration."`
+			choice
+				`	"Do the Gegno upset you in some way?"`
+					goto sacrifice
+			label guidance
+			`	The Quarg now shifts facial expressions in such a way to suggest a small hint of frustration. "The Gegno believe it to be necessary to overwhelmingly sacrifice whatever must be, in order to imagine a grand future for themselves. Thus, they have separated into two different branches. One brandishes themselves as the honorable Vi - mightiest of warriors, toughest of kin. They are brutes that see proof of strength being the one and only variable in determining their path. That of the Scin, the more technologically able, believe that only through progression of sciences and discovery, will the Gegno prosper."`
+				goto next
+			label sacrifice
+			`	"The Gegno believe it to be necessary to sacrifice whatever must be, in order to imagine a future for themselves. Thus, they have separated into two different branches. One brandishes themselves as the honorable Vi - mightiest of warriors, toughest of kin. They are brutes that see proof of strength being the one and only variable in determining their path. That of the Scin, the more technologically able, believe that only through progression of sciences and discovery, will the Gegno prosper."`
+
+			label next
+			choice
```

Comment:
It was kind of a way to give the player('s pilot) an option to choose which faction they seem to relate to more in terms of ideology, since a good portion of the story will be centered around exploring both sides' objectives. Is there a better way you suggest?